se_denseslam/src/preprocessing.cpp

```diff
@@ -31,16 +31,16 @@
  SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
  CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
  OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
- OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE. 
+ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
-#include "timings.h"
-#include <se/utils/math_utils.h>
 
-#include <functional>
-#include <se/image/image.hpp>
+#include <se/preprocessing.hpp>
 
-void bilateralFilterKernel(se::Image<float>& out, const se::Image<float>& in,
-		const std::vector<float>& gaussian, float e_d, int r) {
+void bilateralFilterKernel(se::Image<float>&         out,
+                           const se::Image<float>&   in,
+		                   const std::vector<float>& gaussian,
```

Comment:
indent

Reply:
Fixed indentation and converted tabs to spaces where needed.